### ISSUE TITLE
Versioning of CMSIS Core missmatch

### ISSUE THREAD

**Issue by JavierReyes945** [ARM-software/CMSIS_5#1638]
The latest release announced in the repository is the version 5.9.0

Some notes on the previous release 5.8.0 show that there were issues with the version shown in the `cmsis_version.h`.

At this very moment, the version shown in `cmsis_version.h` for the release 5.9.0 and newer commits show:

```c
/**************************************************************************//**
 * @file     cmsis_version.h
 * @brief    CMSIS Core(M) Version definitions
 * @version  V5.0.5
 * @date     02. February 2022
 ******************************************************************************/
```

```c
/*  CMSIS Version definitions */
#define __CM_CMSIS_VERSION_MAIN  ( 5U)                                      /*!< [31:16] CMSIS Core(M) main version */
#define __CM_CMSIS_VERSION_SUB   ( 6U)                                      /*!< [15:0]  CMSIS Core(M) sub version */
#define __CM_CMSIS_VERSION       ((__CM_CMSIS_VERSION_MAIN << 16U) | \
                                   __CM_CMSIS_VERSION_SUB           )       /*!< CMSIS Core(M) version number */
#endif
```

Which version is supposed to be the CMSIS Core version? Is this an issue with outdated info in `cmsis_version.h`, or does CMSIS have some other versioning system and what the header shows is not exactly the version of the whole release?

**Comment by JonatanAntoni** [comment]
Hi @JavierReyes945,

I appreciate this is a bit confusing. The CMSIS Release version 5.9.0 relates to the bundle while the cmsis_version.h from CMSIS-Core relates to the component version, which is 5.6.0.

Cheers,
Jonatan

**Comment by JavierReyes945** [comment]
> Hi @JavierReyes945,
> 
> I appreciate this is a bit confusing. The CMSIS Release version 5.9.0 relates to the bundle while the cmsis_version.h from CMSIS-Core relates to the component version, which is 5.6.0.
> 
> Cheers, Jonatan

Thank you for the clarification. It would be adviceable to document this somewhere, perhaps the main README.md.

**Comment by JonatanAntoni** [comment]
Good catch. Perhaps we enhance this for CMSIS 6 here: https://github.com/ARM-software/CMSIS_6.
Would you add a section that reflects your understanding?
